test/hotspot/jtreg/serviceability/sa/ClhsdbFindPC.java

```diff
@@ -49,6 +49,7 @@
  * @requires vm.compMode != "Xcomp"
```

Comment:
The summary says "with Xcomp" but the requires says "!= Xcomp" ???

Reply:
Xcomp is in reference to how the debuggee is launched, and that is controlled by the test. See next comment for more.